core/entities/models/JsonModelSchema.php

```diff
@@ -129,15 +178,71 @@ public function getInitialSchemaStructure()
             '$schema'    => 'http://json-schema.org/draft-04/schema#',
             'title'      => $this->model->get_this_model_name(),
             'type'       => 'object',
-            'properties' => array(),
+            'properties' => array(
```

Comment:
I realized I added `link`, `_links`, and `_calculated_fields` to the wrong index on the schema array and they needed to be nested in the top-level `properties` element.  Hence, the change here.